src/mscorlib/shared/System/Globalization/TextInfo.cs

```diff
@@ -258,6 +258,16 @@ private unsafe string ToLowerAsciiInvariant(string s)
             }
         }
 
+        internal void ToLowerAsciiInvariant(ReadOnlySpan<char> source, Span<char> destination)
+        {
+            Debug.Assert(destination.Length >= source.Length);
+
+            for (int i = 0; i < source.Length; i++)
+            {
+                destination[i] = ToLowerAsciiInvariant(source[i]);
```

Comment:
Were you able to confirm that ToLower/UpperAsciiInvariant gets inlined?  Seems like it should, but it'd be good to double check.